DX-1845: One Click Payments issues in payments only (#1695)
SwedbankPay/developer.swedbankpay.com: _includes/one-click-payments.md

Authorization: Bearer <AccessToken>
```

{% else %}

> When the initial purchase is successful, a `paymentToken` is linked to
> the payment. You can return the value by sending a `GET` request towards the
> payment resource with the `payerReference` included.

Reviewer: This encourages clients to build the URL to the token. Can't the URL be discovered in a previous response, somehow?

```suggestion
When the initial purchase is successful, a `paymentToken` is linked to
the payment. You can return the value by performing a `GET` request towards the
payment resource with the `payerReference` included.
```

{:.code-view-header}
**Request Towards The Payment Resource**

```http

> GET /psp/paymentorders/payerownedtokens/{payerReference} HTTP/1.1

Reviewer: ```suggestion
GET /psp/paymentorders/payerownedtokens/{{ page.payment_token }} HTTP/1.1
```

Authorization: Bearer <AccessToken>
```

> You can also send a GET request towards the `paymentOrderId` and find the
> paymentToken in the `paid` resource.

Reviewer: ```suggestion
You can also perform a GET request towards the `id` of a Payment Order and find the
paymentToken in its linked `paid` resource.
```

{:.code-view-header}

> **Request Towards The Payment Resource**

Reviewer: Before performing the request towards the `paid` resource, we should show how the URL to it can be found within a payment order.

```suggestion
**Request Towards The Paid Resource**
```

```http
GET /psp/paymentorders/{{ page.payment_id }}/paid HTTP/1.1
Authorization: Bearer <AccessToken>
```

> 

Reviewer: I think we need to provide an example of how the `paid` resource looks like.

Reviewer: @arebra, this is still unaddressed. ☝🏼

Author: I will address it tomorrow :)
